Octo-Bot/packet-generator-bot/bot/PacketGenerator.py

```diff
@@ -0,0 +1,114 @@
+import time
+
+from scapy.all import *
+
+
+class PacketGenerator():
+
```

Comment:
Can you a little documentation of each function?

Reply:
I have added documentation to most of the main functions!

Reply:
Thank @krusagiz , I will make as `resolved`